### PR TITLE
Move curriculum marketing pages into /curriculum directory

### PR DESCRIPTION
**Pull request by kelbyhawn**
Redirects pages in the `/educate/curriculum` directory into the `/curriculum` directory:
- [code.org/blockchain](http://code.org/blockchain) → [code.org/curriculum/blockchain](http://code.org/curriculum/blockchain) 
- [code.org/educate/csa](https://code.org/educate/csa) → [code.org/curriculum/csa](http://code.org/curriculum/csa)
- [code.org/educate/csp](https://code.org/educate/csp) → [code.org/curriculum/csp](http://code.org/curriculum/csp)
- [code.org/educate/csd](https://code.org/educate/csd) → [code.org/curriculum/csd](http://code.org/curriculum/csd)
- [code.org/educate/csc](https://code.org/educate/csc) → [code.org/curriculum/csc](http://code.org/curriculum/csc)
- [code.org/educate/csf](https://code.org/educate/csf) → [code.org/curriculum/csf](http://code.org/curriculum/csf)
- [code.org/educate/curriculum/csf-transition-guide](https://code.org/educate/curriculum/csf-transition-guide) →  [support.code.org/hc/en-us/articles/26001058366093-Teaching-Computer-Science-Fundamentals-Courses-A-F](https://support.code.org/hc/en-us/articles/26001058366093-Teaching-Computer-Science-Fundamentals-Courses-A-F)
- [code.org/educate/curriculum/elementary-school](https://code.org/educate/curriculum/elementary-school) → [code.org/curriculum/elementary-school](http://code.org/curriculum/elementary-school)
- [code.org/educate/curriculum/high-school](https://code.org/educate/curriculum/high-school) → [code.org/curriculum/high-school](http://code.org/curriculum/high-school)
- [code.org/educate/curriculum/middle-school](https://code.org/educate/curriculum/middle-school) → [code.org/curriculum/middle-school](http://code.org/curriculum/middle-school)
- [code.org/educate/curriculum/values](https://code.org/educate/curriculum/values) → [code.org/curriculum/values](http://code.org/curriculum/values)
- [code.org/educate/curriculum/3rd-party](https://code.org/educate/curriculum/3rd-party) → [code.org/curriculum/3rd-party](http://code.org/curriculum/3rd-party)
- [code.org/ai/how-ai-works](https://code.org/ai/how-ai-works) → [code.org/curriculum/how-ai-works](http://code.org/curriculum/how-ai-works)

**Jira ticket:** [ACQ-1539](https://codedotorg.atlassian.net/browse/ACQ-1539)

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/educate/csp.redirect [hunk 1]
```diff
@@ -0,0 +1 @@
+https://code.org/curriculum/csp
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/educate/csp.redirect:1]: There is some weird legacy stuff going on with the relative URL so I pointed this to the absolute url to make it work.

**Comment by dmcavoy** [comment]
Does this live in that directory too? https://code.org/educate/curriculum/3rd-party

**Comment by dmcavoy** [comment]
Can you also move https://code.org/ai/how-ai-works to code.org/curriculum/how-ai-works?

**Comment by kelbyhawn** [comment]
@dmcavoy just moved those over

**Review by TurnerRiley — APPROVED** [review]
LGTM!

**Pull request merged**